Spec/RestClientPresence.swift

```diff
@@ -75,9 +75,42 @@ class RestClientPresence: QuickSpec {
                     }
                 }
 
+                // RSP3a2
+                pending("clientId should filter members by the provided clientId") {
+                    let options = AblyTests.commonAppSetup()
+                    let client = ARTRest(options: options)
+                    let channel = client.channels.get("test")
+
+                    var realtime = ARTRealtime(options: options)
+                    defer { realtime.close() }
+                    let realtimeChannel = realtime.channels.get("test")
+
+                    realtimeChannel.presence.enterClient("ana", data: "mobile")
+                    realtimeChannel.presence.enterClient("john", data: "desktop")
+                    realtimeChannel.presence.enterClient("john", data: "mobile")
+                    realtimeChannel.presence.enterClient("casey", data: "web")
+                    realtimeChannel.presence.enterClient("casey", data: "mobile")
+                    realtimeChannel.presence.enterClient("john", data: "web")
```

Comment:
I was checking if the member was updating the data and I forgot to remove it. Fixed 31f0591.